Services/TestedService.cs

```diff
@@ -52,14 +52,8 @@ public Task<ContentItem> GetContentItemOrThrowAsync(string id)
     private async Task<ContentItem> GetContentItemOrThrowInternalAsync(string id)
     {
         // You already know how this works :).
-        var contentItem = await _contentManager.GetAsync(id);
-
-        // Checking content retrievals for null is always a good idea.
-        if (contentItem == null)
-        {
-            throw new InvalidOperationException($"The content item with the ID {id} doesn't exist.");
-        }
-
+        var contentItem = await _contentManager.GetAsync(id)
+            ?? throw new InvalidOperationException($"The content item with the ID {id} doesn't exist.");
         return contentItem;
     }
```

Comment:
I think we can drop the `contentItem` variable altogether now:

```suggestion
    private async Task<ContentItem> GetContentItemOrThrowInternalAsync(string id) =>
        // You already know how this works :).
        await _contentManager.GetAsync(id)
        ?? throw new InvalidOperationException($"The content item with the ID {id} doesn't exist.");
```

Reply:
I get this here but I added your suggestion.

![image](https://github.com/Lombiq/Orchard-Training-Demo-Module/assets/1976647/10fa154d-6d79-4129-bfab-c332721aac9b)